tests/Imagine/Cache/Resolver/AwsS3ResolverTest.php

```diff
@@ -12,11 +12,11 @@
 namespace Liip\ImagineBundle\Tests\Imagine\Cache\Resolver;
 
 use Aws\S3\S3Client;
-use Guzzle\Service\Resource\Model;
 use Liip\ImagineBundle\Imagine\Cache\Resolver\AwsS3Resolver;
 use Liip\ImagineBundle\Imagine\Cache\Resolver\ResolverInterface;
 use Liip\ImagineBundle\Model\Binary;
 use Liip\ImagineBundle\Tests\AbstractTest;
+use Liip\ImagineBundle\Tests\Fixtures\Model;
```

Comment:
if the aws client ever starts to declare return types, this will not match the return type, right? i'd hope they only do that in a new major version so i don't expect it to happen soon or uncontrolled.

but as we don't use the return type, can't we simply remove mocking the return from the S3Client in this test?